src/Machine.Specifications.Runner.VisualStudio/Discovery/BuiltIn/TestDiscoverer.cs

```diff
@@ -14,7 +14,13 @@ public class TestDiscoverer
                                 : MarshalByRefObject
 #endif
     {
-
+#if !NETSTANDARD
+        [System.Security.SecurityCritical]
```

Comment:
Add to `usings` to shorten this

Reply:
I added it here to avoid people removing the unused using
![image](https://user-images.githubusercontent.com/1986660/79437311-eaa41f80-7fc9-11ea-99b3-a1b2681a11b8.png)